hw/top_earlgrey/lint/top_earlgrey.waiver

```diff
@@ -25,6 +25,10 @@ waive -rules CLOCK_MUX -location {clkmgr.sv top_earlgrey.sv} -regexp {.*clk_io_d
 waive -rules {COMBO_LOOP} -location {chip_earlgrey_asic.sv} -regexp {'tx' driven in module 'uart_core' by 'rx' at uart_core.sv} \
       -comment "there is technically a loopback path through uart, however RX / TX should never be configured to the same pin"
 
+# Combo loops through sysrst_ctrl inversion can be ignored (TODO: need to double check, are the inversions meant to be direct?)
```

Comment:
@esshiu

Reply:
I traced the cio_key2_out_o logic. It goes through the inversion logic(sysrst_ctrl_inv.sv), the override logic(sysrst_ctrl_pin.sv) and autobloack logic(sysrst_ctrl_autoblock.sv). It's indeed possible to have the passthrough path from cio_key2_in_i to cio_key2_out_o. 

The feature is described in the one paper:
At least 4 input pins with hardware passthrough to 4 matching output pins.
Nominally assigned to power button, keyboard column, 2 keyboard rows

Reply:
thanks @esshiu !